src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -1,14 +1,17 @@
 <%@ tag description="instructorFeedbacks and instructorHome - Feedback Session publish/unpublish button" %>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
+<%@ attribute name="showButtonAndTooltip" type="java.lang.Boolean" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
-   href="${publishButton.actionLink}"
-   title="${publishButton.tooltipText}"
-   data-toggle="tooltip"
-   data-placement="top"
-   data-fsname="${publishButton.feedbackSessionName}"
+<a href="${publishButton.actionLink}" 
+   class="<c:if test="${showButtonAndTooltip}">${buttonType}</c:if> session-${publishButton.actionNameLowercase}-for-test"
```

Comment:
I think she meant to write the line -

``` java
class="<c:if test="${showButtonAndTooltip}">${buttonType}</c:if> session
```

as -

``` java
class="<c:if test="${showButtonAndTooltip}">${buttonType} </c:if>session
```

So that when `c:if` fails it will look like this (no leading space) -

``` html
class="session-..."
```

instead of -

``` html
class=" session-..."
```